Makefile

```diff
@@ -282,8 +285,13 @@ build/src/libultra/libc/llcvt.o: src/libultra/libc/llcvt.c
 	@$(OBJDUMP) -d $@ > $(@:.o=.s)
 
 # Build C files from assets
+
+assets/%.c: assets/%.xml
+	$(ZAPD) bsf -eh -i $< -o $(dir $<)
+	$(CC) -c $(CFLAGS) $(MIPS_VERSION) $(OPTFLAGS) -o build/$(@:.c=.o) $@
```

Comment:
Is this necessary?

Reply:
It re-extracts if the xml is touched, which is present in OoT, so I figured it would be good to bring over.

Reply:
But assets aren't being extracted by running `make`, those are being extracted by `extract_assets,py`.
I feel like that is something old that nobody has deleted in OoT

Reply:
The point is that this rule does extract by running `make`, if any xmls have changed since last setup. I think this can be useful in object decomp. As for whether it's old in OoT I have no idea, and someone from OoT will need to chime in on this and clarify the position of this make rule.

Reply:
In all the time I worked in object decomp for OoT it never worked, I always needed to run `extract_assets.py`
Also, the current `extract_assets.py` only extract XML that has been changed since last extraction, so I don't see the point.